formats/developers/compression.txt

```diff
@@ -0,0 +1,170 @@
+File compression
+================
+
+This section provides an analysis of several file formats (including 
+:doc:`OME-XML </ome-xml/index>` and :doc:`OME-TIFF </ome-tiff/index>`) and 
+compression techniques.
+
+The figures regarding various storage formats were computed from the :ref:`tubhiswt_samples` before the current schema version. Thus, the byte counts between the downloadable ZIP files and the "zipped OME-TIFF" entry do not precisely match.  The table below also lists the space
+requirements for each dataset with various formats and compression
+types.
+
+
```

Comment:
For this entire section, I think it would be more useful for the reader if we represented the numbers as the percentage of the original file size, so that it's clear what the original size was (OME-XML, OME-TIFF), and what the compression efficiency was.  The user isn't going to care about the byte size here either, so I think it would be OK to express it in kilo or mega bytes--it won't detract from the message we're getting across.  I'm also seeing wrapping of the last column, so reducing the information to the minimum needed would also help there if the table formatting doesn't.